routes/x/module.tsx

```diff
@@ -728,6 +741,35 @@ function InfoView(
             </div>
           </div>
 
+          {Object.keys(dependencies).length !== 0 && (
+            <div class="space-y-2">
+              <div class="text-gray-400  text-sm leading-4">
+                Dependencies
+              </div>
+              {Object.entries(dependencies).sort(([kindA], [kindB]) =>
+                kindA.localeCompare(kindB)
+              ).map(([kind, dependencies]) => (
+                <div>
+                  <>
+                    <div class="font-medium">{kind}</div>
+                    <div class="children:(inline-block max-w-full mr-2 link truncate)">
```

Comment:
<img width="350" alt="スクリーンショット 2022-12-08 14 31 09" src="https://user-images.githubusercontent.com/613956/206364566-485055ac-6c9d-4480-844d-8249b982e898.png">

When I first see this, I missed `ts_morph@16.0.0` existing (it looked like a part of version name of `rutt@..`. How about making this `children:block` instead of `children:inline-block` and list all deps vertically?